src/pages/beneficiaryDetails/donateForm/DonerInformationForm.tsx

```diff
@@ -45,15 +51,10 @@ export default function DonerInformationForm({
     if (isLoading === "completed") {
       setIsConinue(true);
     }
-  }, [isLoading]);
+  }, [isLoading,setIsConinue]);
```

Comment:
Noting a lack of spacing between array members; linter or prettier should correct this.